fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -280,13 +277,9 @@ public Response deleteObject() {
 
         try {
             resource().delete();
-
-            try {
-                session.save();
-            } catch (final RepositoryException e) {
-                throw new RepositoryRuntimeException(e);
+            if (!batchService.exists(session.getId(), getUserPrincipal())) {
```

Comment:
@acoburn, thanks for giving this some thought. Are you planning on the "forthcoming commit" to be included in this PR or a subsequent PR?